contracts/true-currencies/TokenController.sol

```diff
@@ -86,6 +86,8 @@ contract TokenController {
     // pausing the contract upgrades the proxy to this implementation
     address public constant PAUSED_IMPLEMENTATION = 0x3c8984DCE8f68FCDEEEafD9E0eca3598562eD291;
```

Comment:
I just noticed there exists this other pause semantic.

Could we rename pause/unpause PoR to enable/disable PoR? This way there's no possible confusion between pausing TUSD and pausing PoR on TUSD.

Reply:
Good point about the inconsistent naming! It is an effect of discussing the feature with various people: operations people preferred `pause`, whereas when discussing it with engineers `proofOfReserveEnabled` seemed a more natural name of a variable that needs to be used to implement the feature. To be consistent I will also rename `proofOfReservePauser` to `proofOfReserveEnabler`, if that's OK.

Reply:
Renamed "pause/unpause" to "enable/disable"